app/src/main/java/chat/rocket/android/model/ServerConfig.java

```diff
@@ -31,6 +32,7 @@
   private int state;
   private String session;
   private String error;
+  private boolean syncPushToken;
```

Comment:
ServerConfig is originally designed to represent(sync) the connectivity of each server.

Is `syncPushToken` related with the connectivity? If not, this property should be in another RealmObject-based class.